### PR TITLE
Disable "Commit" button in VCS plugin if there's no commit message

### PR DESCRIPTION
**Pull request by YeldhamDev**

### CODE REVIEW

File: editor/plugins/version_control_editor_plugin.cpp [hunk 1]
```diff
@@ -294,6 +288,10 @@ void VersionControlEditorPlugin::_update_commit_status() {
 	staged_files_count = 0;
 }
 
+void VersionControlEditorPlugin::_update_commit_button() {
+	commit_button->set_disabled(commit_message->get_text() == "");
```
Comment (akien-mga) [inline editor/plugins/version_control_editor_plugin.cpp:292]:
```suggestion
	commit_button->set_disabled(commit_message->get_text().strip_edges() == "");
```

**Comment by YeldhamDev** [comment]
@akien-mga Updated.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.4.

**Pull request merged**